src/fixtures/mock_ui_state.js

```diff
@@ -15,6 +15,7 @@ export default {
   emit: _.noop,
   on: _.noop,
   off: _.noop,
+  clearAllKeys: _.noop,
```

Comment:
I think this method, should just to the same as `_reset`, so it would kind of simulate uiState better.